app/src/main/java/com/gazlaws/codeboard/CodeBoardIME.java

```diff
@@ -67,6 +67,12 @@ public class CodeBoardIME extends InputMethodService
     private KeyboardUiFactory mKeyboardUiFactory = null;
     private KeyboardLayoutView mCurrentKeyboardLayoutView = null;
 
+    /* Added by Dimitris Stoukogiannis (aka ShadyGeek) on 14/2/2023.
```

Comment:
Nit: for declaration of a variable, we don't need the logic that's going to happen in the code.

I would remove this whole comment - any modern IDE will show the git user who added this line, so adding your name in the code would just be visual clutter. Of course, feel free to add your name to the README if you like! Add a new section "Contributors".

Reply:
Yes I know both were unnecessary, I just added the commetns so that you could see what where the changes I made and why made them and because I knew you could just cross them out before merging the project.

Reply:
Ah, got it! 👍